src/commands/subscriptions-params-manage.js

```diff
@@ -6,13 +6,13 @@ const describe = 'Change Subscriptions parameters'
 
 const builder = {
   output: { alias: 'o', describe: 'Output dir', type: 'string', default: './data/output' },
-  config: { alias: 'c', describe: 'Management config JSON file', type: 'string', default: './data/config/susbcriptions-params-management.js' },
-  feeToken: { describe: 'Set ', type: 'boolean', default: false },
-  feeAmount: { describe: 'Set ', type: 'boolean', default: false },
-  prePaymentPeriods: { describe: 'Set ', type: 'boolean', default: false },
-  latePaymentPenaltyPct: { describe: 'Set ', type: 'boolean', default: false },
-  governorSharePct: { describe: 'Set ', type: 'boolean', default: false },
-  resumePrePaidPeriods: { describe: 'Set ', type: 'boolean', default: false },
+  config: { alias: 'c', describe: 'Management config JSON file', type: 'string', default: './data/config/subscriptions-params-management.js' },
+  feeToken: { describe: 'Set ', type: 'string', default: false },
```

Comment:
Why are we changing these to strings? And in that case, shouldn't the default value change too?

Reply:
I think there was a confusion here, what was the intention here then?

Reply:
See discussion below, the idea was to have boolean flags, that signal which setter you want to call. Then you get the values from the config file.

Reply:
Answered below